### PR TITLE
Develop to main 

### PR DESCRIPTION
**Pull request by anzau**

### CODE REVIEW

**Review by Jo-Zh — APPROVED** [review]
LGFM

File: src/components/MCQ/MCQQuiz.vue [hunk 1]
```diff
@@ -3,34 +3,58 @@
...
     :options-list="currentQuestion.optionsList"
+    :_id="currentQuestion._id"
     @next-question="nextQuestion"
     @skip-question="skipQuestion"
   />
-  <div v-if="!currentQuestion">You are done! Please refresh this page.</div>
+  <MCQStatus
+    v-if="!currentQuestion"
+    :quiz-status="questionsQueue.quizStats"
+    :work-quiz="questionsQueue.quizStats.length"
```
Comment (Jo-Zh) [inline src/components/MCQ/MCQQuiz.vue:13]: probably can delete these props just use questionsQuene.quizStats directly in MCQStatus. in another code refactoring ticket or?

File: src/components/MCQ/MCQOption.vue [hunk 2]
```diff
@@ -3,11 +3,12 @@
   <input
     :id="'option-' + optionKey"
     :key="optionKey"
+    test-id="radio_options"
```
Comment (Jo-Zh) [inline src/components/MCQ/MCQOption.vue:6]: sorry, this line not used in tests, forgot to delete....

Comment (anzau) [reply src/components/MCQ/MCQOption.vue:6]: I can do this one

File: src/components/MCQ/MCQQuiz.vue (hunk 1 above)
Comment (anzau) [reply src/components/MCQ/MCQQuiz.vue:13]: This one should be part of a refactoring, I can only change anything minor at this level but once we start dealing with multiple changes such as props and types, it should be part of a ticket.

**Pull request merged**